### PR TITLE
fix: ListViewItem style glyph similar as in CheckBox

### PR DESCRIPTION
**Pull request by limeniye**
GitHub Issue (If applicable): closes https://github.com/unoplatform/uno/issues/13564

## PR Type

The glyph of ListViewItem is not displayed.

## What is the current behavior?

ListViewItems

## What is the new behavior?

Screenshot 2023-09-08 114048

### <samp>🤖 Generated by Copilot at ff1be7a</samp>

Updated the `MultiSelectCheck` icon to use a custom symbol in `Generic.xaml`. This improves the cross-platform consistency and design of the `ListView` control.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: src/Uno.UI/UI/Xaml/Style/Generic/Generic.xaml [hunk 1]
```diff
@@ -1393,8 +1393,7 @@
 							</Border.RenderTransform>
 							<FontIcon x:Name="MultiSelectCheck"
 									  FontFamily="{ThemeResource SymbolThemeFontFamily}"
-									  win:Glyph="&#xE73E;"
-									  not_win:Glyph="&#xE081;"
```
Comment (Youssef1313) [inline src/Uno.UI/UI/Xaml/Style/Generic/Generic.xaml:1397]: I think E73E should be the correct glyph.

File: src/Uno.UI/UI/Xaml/Style/Generic/Generic.xaml [hunk 2]
```diff
@@ -1393,8 +1393,7 @@
...
+									  Glyph="&#xE001;"
```
Comment (Youssef1313) [inline src/Uno.UI/UI/Xaml/Style/Generic/Generic.xaml:1396]: You'll need to also change it for GridViewItem (in the same file)

File: src/Uno.UI/UI/Xaml/Style/Generic/Generic.xaml (hunk 1 above)
Comment (limeniye) [reply src/Uno.UI/UI/Xaml/Style/Generic/Generic.xaml:1397]: Yes, you are right. But I don't understand why the default CheckBox style has a different Glyph In contrast to ListViewItem.
Does this make sense?

File: src/Uno.UI/UI/Xaml/Style/Generic/Generic.xaml (hunk 2 above)
Comment (limeniye) [reply src/Uno.UI/UI/Xaml/Style/Generic/Generic.xaml:1396]: Already changed there https://github.com/unoplatform/uno/pull/13349

File: src/Uno.UI/UI/Xaml/Style/Generic/Generic.xaml (hunk 1 above)
Comment (limeniye) [reply src/Uno.UI/UI/Xaml/Style/Generic/Generic.xaml:1397]: Done

Comment (Youssef1313) [reply src/Uno.UI/UI/Xaml/Style/Generic/Generic.xaml:1397]: Both E73E and E001 appears to be exactly the same glyph in Uno.Fonts.Fluent package. So both should work. But E73E would keep parity with Windows.`

**Review by Youssef1313 — APPROVED** [review]
LGTM

File: src/Uno.UI/UI/Xaml/Style/Generic/Generic.xaml (hunk 2 above)
Comment (limeniye) [reply src/Uno.UI/UI/Xaml/Style/Generic/Generic.xaml]: Done!

File: src/Uno.UI/UI/Xaml/Style/Generic/Generic.xaml (hunk 1 above)
Comment (limeniye) [reply src/Uno.UI/UI/Xaml/Style/Generic/Generic.xaml:1397]: Done!
By the way it looks better than was.
Before that, they were larger.
image

File: src/Uno.UI/UI/Xaml/Controls/MenuFlyout/MenuFlyout_rs5_themeresources.xaml [hunk 3]
```diff
@@ -398,8 +398,7 @@
                             </Grid.ColumnDefinitions>
                             <FontIcon x:Name="CheckGlyph"
                                 FontFamily="{ThemeResource SymbolThemeFontFamily}"
-                                win:Glyph="&#xE001;"
-                                not_win:Glyph="&#xE10B;"
+                                Glyph="&#xE73E;"
```
Comment (Youssef1313) [inline src/Uno.UI/UI/Xaml/Controls/MenuFlyout/MenuFlyout_rs5_themeresources.xaml:401]: For this one specifically, let's keep E001 to keep parity with Windows.

**Comment by limeniye** [comment]
https://github.com/unoplatform/uno/issues/13564

**Review by jeromelaban — APPROVED** [review]
Thanks for your contribution! Indeed, this particular glyph was using the previous version of the font, and it's only affecting the samples app at this time.

**Review by ajpinedam — APPROVED** [review]

**Pull request merged**